src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/config/ErrorConfiguration.kt

```diff
@@ -94,16 +96,33 @@ class ErrorConfiguration(private val telemetryClient: TelemetryClient) {
     return errorResponse(HttpStatus.CONFLICT, "entity already exists", e.message)
   }
 
+  @ExceptionHandler(WebClientRequestException::class)
+  fun handleWebClientRequestException(e: WebClientRequestException): ResponseEntity<ErrorResponse> {
+    logger.info("web client request exception", e)
+    return errorResponse(HttpStatus.INTERNAL_SERVER_ERROR, "web client request exception", e.message, userMessageForWebClientException(HttpStatus.INTERNAL_SERVER_ERROR))
+  }
+
   @ExceptionHandler(WebClientResponseException::class)
   fun handleWebClientResponseException(e: WebClientResponseException): ResponseEntity<ErrorResponse> {
-    logger.info("web client exception", e)
-    return errorResponse(e.statusCode, "web client exception", e.responseBodyAsString)
+    logger.info("web client response exception", e)
+    return errorResponse(e.statusCode, "web client response exception", e.responseBodyAsString, userMessageForWebClientException(e.statusCode))
+  }
+
+  fun userMessageForWebClientException(status: HttpStatus?): String? {
+    return when {
+      status == null -> null
+      status == HttpStatus.CONFLICT -> null
+      status.is4xxClientError -> "A problem has been encountered. Please contact Support"
+      status.is5xxServerError -> "Delius is experiencing issues. Please try again later and if the issue persists contact Support"
```

Comment:
If we have more calls outside community-api that raises this exception, we'd label everything as a Delius problem. Is there a source that can be added here instead?

Also, this feels like a UI concern based on a triplet of ["dependency call failed", "community-api", 503]